pulseapi/entries/views.py

```diff
@@ -26,77 +29,62 @@
 from pulseapi.utility.userpermissions import is_staff_address
 
 @api_view(['PUT'])
+@permission_classes([IsAuthenticated])
```

Comment:
I'm not a fan of this solution: we are not checking whether a user has a permission, we are checking whether a user is authenticated or not, so let's not do this. Let's keep using `user.is_authenticated()`